app/views/energy_tariffs/energy_tariffs/new.html.erb

```diff
@@ -4,30 +4,12 @@
   <%= render 'energy_tariffs/energy_tariffs/meters', energy_tariff: @energy_tariff if @energy_tariff.tariff_holder_type == 'School' %>
   <br/>
 
-  <% if @energy_tariff.tariff_holder_type == 'School' %>
-    <%= simple_form_for [@school, @energy_tariff] do |f| %>
-      <%= f.hidden_field :meter_type, value: @energy_tariff.meter_type %>
-      <%= f.hidden_field :tariff_type, value: @energy_tariff.tariff_type %>
-      <% @energy_tariff.meters.each do |meter| %>
-        <%= f.hidden_field :meter_ids, value: meter.id, multiple: true %>
-      <% end %>
-      <%= render 'form', school: @school, energy_tariff: @energy_tariff, f: f %>
-    <% end %>
-  <% end %>
-
-  <% if @energy_tariff.tariff_holder_type == 'SchoolGroup' %>
-    <%= simple_form_for [@school_group, @energy_tariff] do |f| %>
-      <%= f.hidden_field :meter_type, value: @energy_tariff.meter_type %>
-      <%= f.hidden_field :tariff_type, value: @energy_tariff.tariff_type %>
-      <%= render 'form', school_group: @school_group, energy_tariff: @energy_tariff, f: f %>
-    <% end %>
-  <% end %>
-
-  <% if @energy_tariff.tariff_holder_type == 'SiteSettings' %>
-    <%= simple_form_for @energy_tariff, url: admin_settings_energy_tariffs_path(@energy_tariff) do |f| %>
-      <%= f.hidden_field :meter_type, value: @energy_tariff.meter_type %>
-      <%= f.hidden_field :tariff_type, value: @energy_tariff.tariff_type %>
-      <%= render 'form', school: @school, energy_tariff: @energy_tariff, f: f %>
+  <%= simple_form_for @energy_tariff.tariff_holder_route + [@energy_tariff] do |f| %>
+    <%= f.hidden_field :meter_type, value: @energy_tariff.meter_type if @energy_tariff.tariff_holder_type == 'School' %>
```

Comment:
meter type and tariff type are required for all tariffs, regardless of holder. Is there  a reason to only include the hidden fields for School tariffs? Just want to make sure we don't end up creating SchoolGroup and SiteSetting tariffs with the default values, rather than those set by the user.

Reply:
👍 have removed the check for school